src/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface NotificationsResponse {
+  notifications: Notification[]
+}
+
+interface Notification {
+  notificationCode: number
+  title: string
+  type: string
```

Comment:
```suggestion
  type: string | null
```
example見る感じだと一応入ることあるっぽい？